packages/desktop/views/dashboard/wallet/panes/account-summary/components/AccountNetworkSummary.svelte

```diff
@@ -109,11 +120,6 @@
 
     account-network-summary-assets {
         @apply px-5 py-4;
-        @apply bg-purple-50;
-    }
-
-    .middle {
-        border-top: 1px solid #f1eef9;
-        border-bottom: 1px solid #f1eef9;
+        @apply bg-surface-1 dark:bg-surface-1-dark;
```

Comment:
```suggestion
        @apply flex flex-row justify-between items-center;
        @apply bg-surface-1 dark:bg-surface-1-dark;
```
+ remove classes on html tag